.gitignore

```diff
@@ -153,7 +159,7 @@ cython_debug/
 #  be found at https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
-#.idea/
+# .idea/
```

Comment:
Didn't you mean to uncomment this? Right now it just adds a space, but the comment character (`#`) is still there.